scripts/release/prod/stage/generate_releases_page/run.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
```

Comment:
Add file header

Reply:
I'm not sure I see the relevance here.  This is script that is is automated and shouldn't isn't invoked on the cli.  The `.generate_releases_page` itself, however, can be called from the cli and therefore does have a header that briefly describes its usage.